src/audio/host.c

```diff
@@ -129,13 +129,31 @@ static uint32_t host_dma_get_split(struct host_data *hd, uint32_t bytes)
 static void host_update_position(struct comp_dev *dev, uint32_t bytes)
 {
 	struct host_data *hd = comp_get_drvdata(dev);
+	struct comp_buffer *source;
+	struct comp_buffer *sink;
+	int ret;
+
 
 	if (dev->direction == SOF_IPC_STREAM_PLAYBACK)
-		dma_buffer_copy_from(hd->dma_buffer, hd->local_buffer,
-				     hd->process, bytes);
+		ret = dma_buffer_copy_from(hd->dma_buffer, hd->local_buffer,
+					   hd->process, bytes);
 	else
-		dma_buffer_copy_to(hd->local_buffer, hd->dma_buffer,
-				   hd->process, bytes);
+		ret = dma_buffer_copy_to(hd->local_buffer, hd->dma_buffer,
+					 hd->process, bytes);
+
+	if (ret < 0) {
```

Comment:
likewise here with our action.

Reply:
Added in https://github.com/thesofproject/sof/pull/3251